public/app/themes/clarity/src/globals/js/auth-heartbeat.js

```diff
@@ -1,6 +1,154 @@
+            button: {
+                continue: {
+                    backgroundColor: 'green',
+                    color: 'white',
+                    cursor: 'pointer'
+                },
+                escape: {
+                    backgroundColor: '#2271b1',
+                    color: 'white',
+                    cursor: 'pointer'
+                }
+            },
+            modal: {
+                textAlign:'left',
+                padding:'20px 30px',
+                background:'#fff',
+                borderRadius:'5px',
+                alignItems: 'center',
+                verticalAlign:'middle',
+                maxWidth:'450px',
+                width: '450px',
+                margin:'50px auto',
+                border: '6px solid #2271b1'
+            },
+            backdrop: {
+                display:'flex',
+                alignItems: 'center',
+                backdropFilter: 'blur(5px)',
+                textAlign:'center',
+                position: 'fixed',
+                zIndex:1000000,
+                height:'100%',
+                width:'100%'
+            }
+        },
+        feature: {
+            can_run: () => {
+                const runOn = [
+                    'dev.intranet',
+                    'intranet.docker',
+                    'staging.intranet'
+                ];
+                let ii = 0;
+
+                for (ii; ii < runOn.length; ii++) {
+                    if (location.href.includes(runOn[ii])) {
+                        return true;
+                    }
+                }
+                return false;
+            }
+        },
+        modal: (title, html) => {
+            let modal = $('<div\>', { 'class': 'heartbeat__modal' })
+                .css(Backdrop.style.modal);
+
+            const backdrop = $('<div\>', { 'class': 'heartbeat__backdrop' })
+                .css(Backdrop.style.backdrop);
+
+            const heading = $('<h3\>').text(title).css({fontWeight:'700', fontSize: '2rem', lineHeight: '3.3rem'})
+            const content = $('<p\>').html(html).css({})
+            const image = $('<img>', {
+                src: '/app/themes/clarity/dist/images/crown_copyright_logo.png',
+                alt: 'Crown copyright logo',
+            }).css(Backdrop.style.img);
+
+            modal = modal.append(image, heading, content);
+            console.log('Heartbeat:', modal);
+
+            return backdrop.append(modal);
+        },
+        feedback: () => {
+            // We can use this to place a box in the bottom right corner of
+            // the screen. This box can remind the user they are UATing and
+            // offer a link to submit feedback.
+            console.log('Display the feedback box.');
+        },
+        confirm: () => {
+            const title = 'You are about to view a test version of the MoJ Intranet'
+            const html = '<br />If you intend to assist in user acceptance testing, please continue.<br /><br />' +
```

Comment:
Just a suggestion, but could you use the word participate instead.

e.g. To participate in user acceptance testing, please continue.